openrtb2/refsettings.go

```diff
@@ -0,0 +1,42 @@
+package openrtb2
+
+import (
+	"encoding/json"
+
+	"github.com/prebid/openrtb/v19/adcom1"
+)
+
+// Object: RefSettings
+//
```

Comment:
Nitpick: i think you can remove this empty comment line

Reply:
I copied the style used in other files of this project, It may look out of place here since the description is short. Would you still like to see this line removed given the context?

Reply:
Oh I see! I think given the context it's fine to leave this line!